x-pack/plugins/maps/public/classes/styles/vector/components/symbol/icon_map_select.tsx

```diff
@@ -44,7 +45,7 @@ interface State {
 
 export class IconMapSelect extends Component<Props, State> {
   state = {
-    customIconStops: this.props.customIconStops ? this.props.customIconStops : DEFAULT_ICON_STOPS,
+    customIconStops: this.props.customIconStops,
```

Comment:
I made `customIconStops` a required prop because setting the `DEFAULT_ICON_STOPS` is now handled [here](https://github.com/elastic/kibana/pull/113144/files#diff-a1f0034f19cd64822a1365f4f491dc899f193a5ec85d0d79244a8e691785a656R1092-R1096)